addons/mail/static/src/messaging/widget/form_renderer/form_renderer.js

```diff
@@ -31,6 +31,13 @@ FormRenderer.include({
         // Do not load chatter in form view dialogs
         this._isFromFormViewDialog = params.isFromFormViewDialog;
     },
+    /**
+     * @override
+     */
+    destroy() {
+        this._super(...arguments);
+        this._destroyChatterContainer();
```

Comment:
Apparently `.destroy()` is automatically called on wrapper when parent `.destroy()` is called.
The only important parts are setting `undefined` on `_chatterContainerComponent` and `.off('o_chatter_rendered')`.

This method should be called only when there is a `chatterContainerComponent`, it's weird to have the `if` condition inside the `_destroyChatterContainer()` function. Also I wouldn't bother with another function if it's called only in `destroy()` and just dump the 2 expressions there inside a `if` conditional branch.